### PR TITLE
Add `Listening` to `events` code exemple

### PR DESCRIPTION
**Pull request by PululuK**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add `Listening` to `events` code exemple
| Fixed ticket? | N/A

### CODE REVIEW

File: src/content/1.7/themes/reference/javascript-events/_index.md [hunk 1]
```diff
@@ -43,6 +43,27 @@ prestashop.emit(
 );
 ```
 
+### Listening to rvents
```
Comment (Progi1984) [inline src/content/1.7/themes/reference/javascript-events/_index.md:46]:
```suggestion
### Listening to events
```

**Review by Progi1984 — COMMENTED** [review]
A small comment

**Review by Progi1984 — APPROVED** [review]

File: src/content/1.7/themes/reference/javascript-events/_index.md [hunk 2]
```diff
@@ -43,6 +43,27 @@ prestashop.emit(
...
+      var eventDatas = {};
+      if (event && event.reason) {
+        eventDatas = {
+          data_form: event.reason.dataForm,
+          product_option: event.reason.productOption
+        };
+      }
+   }
+  );
+}
```
Comment (PierreRambaud) [inline src/content/1.7/themes/reference/javascript-events/_index.md:64]:
```suggestion
if (typeof prestashop !== 'undefined') {
  prestashop.on(
    'product updated',
    function (event) {
      var eventDatas = {};
      if (event && event.reason) {
        eventDatas = {
          data_form: event.reason.dataForm,
          product_option: event.reason.productOption
        };
      }
    }
  );
}
```

**Review by PierreRambaud — APPROVED** [review]

File: src/content/1.7/themes/reference/javascript-events/_index.md [hunk 3]
```diff
@@ -43,6 +43,27 @@ prestashop.emit(
...
 ```
 
+### Listening to events
+
+You can also react to an event emitted by `prestashop.emit`. Here is a simple example:
+
+```js
+if (typeof prestashop !== 'undefined') {
+  prestashop.on(
+    'product updated',
```
Comment (matks) [inline src/content/1.7/themes/reference/javascript-events/_index.md:53]: Line 88 below mentions `updatedProduct` event. Is that a mistake?

Else maybe we should keep both this part and the part after synchronized to help reader?

Comment (PululuK) [reply src/content/1.7/themes/reference/javascript-events/_index.md]: Nice catch @matks !
I just update my PR, its more generic now !

**Review by matks — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by NeOMakinG** [comment]
Thanks @PululuK

**Pull request merged**